index.html

```diff
@@ -10,7 +10,54 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <header class="header">
+        <span>
+            <h1>Weather Report</h1>
+            <h2>For the city of
+                <h3 id="cityname">Seattle</h3>
+            </h2>
+        </span>
+    </header>
+
+    <section class="temperature_section">
+        <h2>Temperature</h2>
+        <div id="tempsection">
+            <div id="changeTemp">
+                <span id="upTempControl">⬆️</span>
+                <span id="temp">69</span>
+                <span id="downTempControl">⬇️</span>
```

Comment:
```suggestion
                <button id="upTempControl">⬆️</button>
                <p id="temp">69</p>
                <button id="downTempControl">⬇️</button>
```
I'd recommend using a `<p>` tag and `<button>` tags here instad of `<span>`s since they have more semantic meaning. `<span>` tags should be using sparingly and only for inline styling.